rnndb/graph/g80_texture.xml

```diff
@@ -112,9 +112,9 @@ xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
 			<value value="8" name="CUBE_ARRAY" />
 		</bitfield>
 		<bitfield pos="18" name="LINEAR" />
-		<bitfield high="21" low="19" name="TILE_MODE_X" />
-		<bitfield high="24" low="22" name="TILE_MODE_Y" />
-		<bitfield high="27" low="25" name="TILE_MODE_Z" />
+		<bitfield high="21" low="19" name="BLOCK_WIDTH" type="block_dimensions" />
```

Comment:
hm, so BLOCK_WIDTH instead of BLOCK_DIMENSIONS_WIDTH? May as well be that way, let's just be consistent.